src/test/java/teammates/test/cases/common/FieldValidatorTest.java

```diff
@@ -55,9 +55,9 @@ public void testGetValidityInfoForSizeCappedNonEmptyString() {
         
         String tooLongName = StringHelper.generateStringOfLength(maxLength + 1);
         assertEquals("invalid: too long",
-                String.format(
-                        SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE,
-                        tooLongName, typicalFieldName, REASON_TOO_LONG, typicalFieldName, maxLength),
+                "\"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\" is not acceptable to TEAMMATES as "
+                + "a/an my field because it is too long. The value of a/an my field should be no longer than 50 "
+                + "characters. It should not be empty.",
```

Comment:
The ones that you just moved here (from being defined as constants) don't have the line break indentations. Don't forget to add them.